sql/moz-fx-data-shared-prod/firefox_desktop/baseline_active_users/view.sql

```diff
@@ -2,62 +2,79 @@ CREATE OR REPLACE VIEW
   `moz-fx-data-shared-prod.firefox_desktop.baseline_active_users`
 AS
 SELECT
-  *,
+  * EXCEPT (
+    app_display_version,
+    normalized_channel,
+    normalized_os,
+    normalized_os_version,
+    windows_build_number
+  ) REPLACE(
+    IFNULL(country, '??') AS country,
+    IFNULL(city, '??') AS city,
+    COALESCE(REGEXP_EXTRACT(locale, r'^(.+?)-'), locale, NULL) AS locale
+  ),
   CASE
-    WHEN LOWER(IFNULL(isp, '')) = 'browserstack'
+    WHEN isp = 'BrowserStack'
       THEN CONCAT('Firefox Desktop', ' ', isp)
-    WHEN LOWER(IFNULL(distribution_id, '')) = 'mozillaonline'
+    WHEN distribution_id = 'MozillaOnline'
       THEN CONCAT('Firefox Desktop', ' ', distribution_id)
     ELSE 'Firefox Desktop'
   END AS app_name,
+  app_display_version AS app_version,
+  normalized_channel AS channel,
+  normalized_os AS os,
+  COALESCE(
+    `mozfun.norm.windows_version_info`(normalized_os, normalized_os_version, windows_build_number),
+    normalized_os_version
+  ) AS os_version_build,
```

Comment:
What's the difference between this and normalized_os_version?

Reply:
I **think**  that normalized_os_version is more or less the same thing that's reported by the client. IIUC, that's not enough to tell windows versions apart for windows 10+. os_version_build derives the "accurate" windows version by referencing the build number.

Reply:
This seems to be windows specific info, should we rename this field to reflect that more?